examples/kafka_message_consumer_service/kafka_message_consumer_service.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/io;
+import ballerina/kafka;
+import ballerina/lang. 'string as strings;
```

Comment:
Better if this is avoided. No need to use an alias. Can refer to functions of lang.string as follows:
```ballerina
'string:fromBytes()
```

Reply:
This looks more like `'` got accidentally typed IMO. Looks like a syntax error.

Reply:
The thing is it will send the wrong message to the user, that you should always use an alias when using lang lib libraries such as `lang.string`, `lang.int` (which is actually how we started using aliases all the time, even though there's no need for it). So as a practice we avoid the use of aliases for lang libraries in our code and docs.

Reply:
We will stick to the alias option since that also a valid syntax.